### PR TITLE
86 add a layout component to wrap pages

### PR DESCRIPTION
**Pull request by Elkrival**
Create App Layout component with Sidebar and Header and refactored chart pages.

### CODE REVIEW

File: views/Chart.react.js [hunk 1]
```diff
@@ -1,18 +1,44 @@
...
-      <h1> Charts</h1>
-      <Button
-        variant="outlined"
-        color="primary"
-        href={routes.newChart}
-      >
-        New Chart
-      </Button>
-    </>
+    <div className={classes.root}>
```
Comment (mikestone14) [inline views/Chart.react.js]: can this div and class go on the `AppLayout`?

File: views/Chart.react.js [hunk 2]
```diff
@@ -1,18 +1,44 @@
...
+      <AppLayout
+        title='Charts'
+        user={user}
```
Comment (mikestone14) [inline views/Chart.react.js]: do we need to pass this or can we get it from redux?

File: views/layouts/AppLayout.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  const toggleDrawer = () => setOpenDrawer(!openDrawer)
+
+  useEffect(() => {
+    fetchSubjects().then(acl => {
+      setSideBarState(getCounts({ acl }))
+    })
+    setAvatar(getAvatar({ user }))
+  }, [])
+
+  return (
```
Comment (mikestone14) [inline views/layouts/AppLayout.jsx:37]: doesn't this need to also render `children`?

Comment (Elkrival) [reply views/layouts/AppLayout.jsx:37]: [3c29fae](https://github.com/AMP-SCZ/dpdash/pull/87/commits/3c29fae6c41739cf5c666b5f0729b2f54fb47adb)

File: views/Chart.react.js (hunk 2 above)
Comment (Elkrival) [reply views/Chart.react.js]: [3c29fae](https://github.com/AMP-SCZ/dpdash/pull/87/commits/3c29fae6c41739cf5c666b5f0729b2f54fb47adb)

File: views/Chart.react.js (hunk 1 above)
Comment (Elkrival) [reply views/Chart.react.js]: [3c29fae](https://github.com/AMP-SCZ/dpdash/pull/87/commits/3c29fae6c41739cf5c666b5f0729b2f54fb47adb)

File: views/Chart.react.js [hunk 4]
```diff
@@ -1,18 +1,25 @@
...
 import { routes } from './routes/routes'
 
-export default function Charts() {
+const Charts = () => {
   return (
-    <>
-      <h1> Charts</h1>
+    <AppLayout
+      title='Charts'
+    >
```
Comment (mikestone14) [inline views/Chart.react.js]:
```suggestion
    <AppLayout title='Charts'>
```

File: views/NewChart.react.js [hunk 5]
```diff
@@ -1,74 +1,25 @@
...
-      <Header
-        handleDrawerToggle={toggleDrawer}
-        title='Create a New Chart'
-        isAccountPage={false}
+    <AppLayout
+      title='Create a Chart'
+    >
+      <ChartForm 
+        handleSubmit={handleSubmit}
       />
```
Comment (mikestone14) [inline views/NewChart.react.js]:
```suggestion
    <AppLayout title='Create a Chart'>
      <ChartForm handleSubmit={handleSubmit} />
```

**Review by mikestone14 — APPROVED** [review]
looks great, just a couple very minor format tweaks.

**Pull request merged**